src/main/java/org/springframework/data/couchbase/core/query/Query.java

```diff
@@ -40,6 +59,15 @@ public Query addCriteria(QueryCriteria criteriaDefinition) {
 		return this;
 	}
 
+	public Query setParameters(JsonValue parameters) {
+		this.parameters = parameters;
+		return this;
+	}
+
+	JsonValue getParameters() {
```

Comment:
it could even be Object so that in the future we can handle lists and maps in addition to JsonObject and JsonArray, right?